stdlib/SparseArrays/docs/src/AbstractSparseMatrixCSC.md

```diff
@@ -0,0 +1,62 @@
+    AbstractSparseMatrixCSC{Tv,Ti<:Integer} <: AbstractSparseMatrix{Tv,Ti}
+
+Supertype for matrix with compressed sparse column (CSC).
+
+!!! compat "Julia 1.4"
+    `AbstractSparseMatrixCSC` requires at least Julia 1.4.
+
+## `AbstractSparseMatrixCSC` interface
+
+In addition to the [Abstract array interface](@ref man-interface-array), every
+`AbstractSparseMatrixCSC` subtype `TS` must provide the following methods:
+
+* [`size(::TS)`](@ref size)
+* [`getcolptr(::TS) :: AbstractVector{<:Ti}`](@ref getcolptr)
+* [`rowvals(::TS) :: AbstractVector{<:Ti}`](@ref rowvals)
+* [`nonzeros(::TS) :: AbstractVector{<:Tv}`](@ref nonzeros)
```

Comment:
In #25118 @simonbyrne suggested `values`. Indeed, it's a bit strange that `nonzeros(A)` can contain zeros.

Unfortunately, `values(::SparseMatrixCSC)` is already a valid method which is defined to be an identity function by the fallback. But maybe it could be considered a "minor change" if we define appropriate `key(::AbstractSparseMatrixCSC)`?

Reply:
As a reference, #22907 is the PR where `keys`, `values` and `pairs` are introduced for arrays.  There seems to be no discussion on the treatment of structured and sparse arrays.

Reply:
So it would be breaking for functions like `findmax` implemented in #22907.

Reply:
I think `values` is a bit generic. Perhaps `storedvals`?

Reply:
`storedvals` sounds like a good idea to me. If we go this way, we can rename `nonzeroinds` for `SparseVector` to `storedindices` as well (ref https://github.com/JuliaLang/julia/pull/33050#discussion_r317329344).